src/components/scrobber/scrobber.js

```diff
@@ -11,15 +12,25 @@ const view = ({completion$, ui}) => completion$
   .startWith(0)
   .map(completion => ui.update(completion))
 
-export default ({completion$, DOM}) => {
-  const ui = new ScrobberUIModel('x-scrobber')
-  const seek$ = DOM
+const model = ({DOM}) => {
+  const dragEnd$ = DOM
     .select(ScrobberUIModel.tagName)
     .events('changeEnd')
+  const dragStart$ = DOM
+    .select(ScrobberUIModel.tagName)
```

Comment:
repeated use of `DOM.select(ScrobberUIModel.tagName)`. I think it could be put in a variable.